packages/wonder-blocks-dropdown/src/components/select-opener.tsx

```diff
@@ -97,74 +114,51 @@ export default class SelectOpener extends React.Component<SelectOpenerProps> {
-                        error,
-                    );
-                    const {hovered, focused, pressed} = state;
-
-                    // The icon colors are kind of fickle. This is just logic
-                    // based on the zeplin design.
-                    const iconColor = light
-                        ? disabled || pressed
-                            ? "currentColor"
-                            : tokens.color.white
-                        : disabled
-                        ? tokens.color.offBlack32
-                        : tokens.color.offBlack64;
+        const stateStyles = _generateStyles(light, isPlaceholder, error);
 
-                    const style = [
-                        styles.shared,
-                        stateStyles.default,
-                        disabled && stateStyles.disabled,
-                        !disabled &&
-                            (pressed
-                                ? stateStyles.active
-                                : (hovered || focused) && stateStyles.focus),
-                    ];
+        // The icon colors are kind of fickle. This is just logic
+        // based on the zeplin design.
+        const iconColor = light
+            ? disabled || error
+                ? "currentColor"
+                : tokens.color.white
+            : disabled
+            ? tokens.color.offBlack32
+            : tokens.color.offBlack64;
 
-                    return (
-                        <StyledButton
-                            {...sharedProps}
-                            aria-disabled={disabled}
-                            aria-expanded={open ? "true" : "false"}
-                            aria-haspopup="listbox"
-                            data-testid={testId}
-                            id={id}
-                            style={style}
-                            type="button"
-                            {...childrenProps}
-                        >
-                            <LabelMedium style={styles.text}>
-                                {/* Note(tamarab): Prevents unwanted vertical
-                                shift for empty selection */}
-                                {children || "\u00A0"}
-                            </LabelMedium>
-                            <PhosphorIcon
-                                icon={caretDownIcon}
-                                color={iconColor}
-                                size="small"
-                                style={styles.caret}
-                                aria-hidden="true"
-                            />
-                        </StyledButton>
-                    );
-                }}
-            </ClickableBehavior>
-        );
-    }
+        const style = [
+            styles.shared,
+            stateStyles.default,
+            disabled && stateStyles.disabled,
+        ];
 
-    render(): React.ReactNode {
         return (
-            <__RouterContext.Consumer>
```

Comment:
There weren't any router related props for SelectOpener so I removed this while removing ClickableBehavior. I'm assuming we had this previously since ClickableBehavior needed a `router`, but let me know otherwise!